screens/Rooms.js

```diff
@@ -19,13 +19,8 @@ class Rooms extends React.Component {
 
   createroom = () => {
     this.props.dispatch(createRoom(this.state.newroom)).then(result => {
```

Comment:
du coup même remarque... ça a déjà fonctionné ? pour moi il faut avoir un mapDispatchToProps qui injecte une fonction qui dispatch l'action !

mais si tu l'as fait partout je suppose que ça fonctionne...

Reply:
bon, quoi qu'il en soit, tu essaies là de faire une action asynchrone (un call API) et t'as besoin de réagir après la fin de l'action

Reply:
pour ça, il te faut un mécanisme supplémentaire : soit les thunks (redux-thunk), soit les saga avec redux-saga

je te recommande le premier pour débuter qui est plus simple (mais saga est beaucoup mieux sur le long terme à mon avis)

voici un article qui a l'air bien https://medium.com/fullstack-academy/thunks-in-redux-the-basics-85e538a3fe60